Packages/Tracking Preview/UI Input/Runtime/Scripts/UIInputModule.cs

```diff
@@ -43,12 +43,14 @@ public class UIInputModule : BaseInputModule, IInputModuleEventHandler
         [SerializeField]
         private Camera mainCamera;
         public Camera MainCamera => mainCamera;
+        public void SetMainCamera(Camera c) { mainCamera = c; }
```

Comment:
Could these not be setters directly on the property above?

Reply:
Changed